routes/web.php

```diff
@@ -24,3 +25,13 @@
     Route::patch('/{city}', 'update');
     Route::delete('/{city}', 'destroy');
 });
+
+Route::group([
+    'prefix' => 'airlines',
+    'controller' => AirlineController::class
+], function () {
+    Route::get('/', 'index');
+    Route::post('/', 'store');
+    Route::delete('/{airline}', 'destroy');
+    Route::patch('/{airline}', 'update');
```

Comment:
You should be careful here. Using a patch is viable, however, you need to take a close look and decide whether it is convenient for you to change your validations or not.